packages/icons/scripts/build.js

```diff
@@ -77,20 +86,22 @@ const writeEnum = (srcDir, iconSetName, idPrefix) => {
   );
 }
 
-if (!existsSync(buildDirPath)) {
-  mkdirSync(buildDirPath);
-}
+[buildDirPath, distDirPath].forEach(buildPath => {
+  mkdirSyncRecursive(buildPath);
```

Comment:
This code works for me:

```suggestion
  !existsSync(buildPath) && mkdirSync(buildPath, { recursive: true });
```

Generating 
```
🔧 "system" icons
	generating sprite at:
	/Users/dima/workspace/@dcos/ui-kit/packages/icons/dist/system-icons-sprite.svg

	optimizing sprite

	generating sprite at:
	/Users/dima/workspace/@dcos/ui-kit/dist/packages/icons/dist/system-icons-sprite.svg

	optimizing sprite

	creating icon name enum

🔧 "product" icons
	generating sprite at:
	/Users/dima/workspace/@dcos/ui-kit/packages/icons/dist/product-icons-sprite.svg

	optimizing sprite

	generating sprite at:
	/Users/dima/workspace/@dcos/ui-kit/dist/packages/icons/dist/product-icons-sprite.svg

	optimizing sprite

	creating icon name enum
```

Reply:
though I'm on v8.11.4

Reply:
on 10.14.2 it also works

Reply:
That's strange. I wonder why it doesn't work on my machine 🤔 

Did you remove the top-level `dist` directory before running `npm run build:svg`?